decidim-proposals/app/jobs/decidim/proposals/notify_proposals_mentioned_job.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Proposals
+    class NotifyProposalsMentionedJob < ApplicationJob
+      def perform(comment, proposal_metadata)
+        # feature = Decidim::Feature.find(feature_id)
+        linked_proposals = proposal_metadata.linked_proposals
+        linked_proposals.each do |proposal|
+          recipient_ids = [proposal.decidim_author_id]
```

Comment:
Remember that a proposal could not have an author

Reply:
I was not aware of it!
@isaacmg410 informed me that it is possible to create official proposals from the Admin panel, in this case decidim_author_id is nil.
:+1: